lib/cryptoexchange/exchanges/gopax/services/market.rb

```diff
@@ -0,0 +1,43 @@
+module Cryptoexchange::Exchanges
+  module Gopax
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            true
+          end
+        end
+
+        def fetch(market_pair)
+          ticker1 = super(ticker_url_one(market_pair))
+          ticker2 = super(ticker_url_two(market_pair))
+          adapt(ticker1, ticker2, market_pair)
+        end
+
+        def ticker_url_one(market_pair)
+          "#{Cryptoexchange::Exchanges::Gopax::Market::API_URL}/trading-pairs/#{market_pair.base}-#{market_pair.target}/ticker"
+        end
+
+        def ticker_url_two(market_pair)
+          "#{Cryptoexchange::Exchanges::Gopax::Market::API_URL}/trading-pairs/#{market_pair.base}-#{market_pair.target}/stats"
```

Comment:
@estherleongym Is it necessary to have 2 calls?
https://api.gopax.co.kr/trading-pairs/ETH-KRW/stats
https://api.gopax.co.kr/trading-pairs/ETH-KRW/ticker

Stats seem to have all the information we need?

Reply:
because stats do not have bid/ask